src/FESpaces/DivConformingFESpaces.jl

```diff
@@ -0,0 +1,103 @@
+# This source file is though to put that code required in order to
+# customize ConformingFESpaces.jl to H(div)-conforming global FE Spaces built
+# out of RaviartThomas FEs. In particular, this customization is in the
+# definition of the shape functions (get_shapefuns) and the DoFs
+# (get_dof_basis) of the **global** FE space, which requires a sign flip
+# for those sitting on facets of the slave cell of the facet.
+
+function get_dof_basis(model::DiscreteModel,
+                       cell_reffe::AbstractArray{<:GenericRefFE{RaviartThomas}})
+    sign_flip = get_sign_flip(model, cell_reffe)
+    lazy_map(get_dof_basis,
+             cell_reffe,
+             get_cell_map(Triangulation(model)),
+             sign_flip)
+end
+
+function get_shapefuns(model::DiscreteModel,
+                       cell_reffe::AbstractArray{<:GenericRefFE{RaviartThomas}})
+    sign_flip = get_sign_flip(model, cell_reffe)
+    lazy_map(get_shapefuns,
+             cell_reffe,
+             get_cell_map(Triangulation(model)),
+             lazy_map(Broadcasting(ConstantField), sign_flip))
+end
+
+function get_sign_flip(model::DiscreteModel,
+                       cell_reffe::AbstractArray{<:GenericRefFE{RaviartThomas}})
+    lazy_map(get_sign_flip,
+            Fill(model, length(cell_reffe)),
+            cell_reffe,
+            get_cell_to_bgcell(model))
+end
+
+function get_sign_flip(model::DiscreteModel,
+                       reffe::GenericRefFE{RaviartThomas},
+                       cell_id::Integer)
+
+    cache = return_cache(get_sign_flip,
+                         reffe,
+                         cell_id)
+    evaluate!(cache,
+              get_sign_flip,
+              reffe,
+              cell_id)
+end
+
+function return_cache(::typeof(get_sign_flip),
+                      model::DiscreteModel,
+                      reffe::GenericRefFE{RaviartThomas},
+                      cell_id::Integer)
+    D = num_cell_dims(model)
+    gtopo = get_grid_topology(model)
+
+    # Extract composition among cells and facets
+    cell_wise_facets_ids = get_faces(gtopo, D, D - 1)
+    cache_cell_wise_facets_ids = array_cache(cell_wise_facets_ids)
+
+    # Extract cells around facets
+    cells_around_facets = get_faces(gtopo, D - 1, D)
+    cache_cells_around_facets = array_cache(cells_around_facets)
+
+    (cell_wise_facets_ids,
+     cache_cell_wise_facets_ids,
+     cells_around_facets,
+     cache_cells_around_facets,
+     CachedVector(Bool))
+end
+
+function evaluate!(cache,
+                   ::typeof(get_sign_flip),
+                  model::DiscreteModel,
+                  reffe::GenericRefFE{RaviartThomas},
+                  cell_id::Integer)
+
+    cell_wise_facets_ids,
+    cache_cell_wise_facets_ids,
+    cells_around_facets,
+    cache_cells_around_facets,
+    sign_flip = cache
+
+    setsize!(sign_flip, (num_dofs(reffe),))
+    sign_flip .= false
```

Comment:
Replace this:
```julia
    sign_flip = cache
    setsize!(sign_flip, (num_dofs(reffe),))
```
by
```julia
    setsize!(cache, (num_dofs(reffe),))
    sign_flip = cache.array
```
Then `sign_flip` (and the return of this function) will be a intrinsic julia array. This is also a common pattern used in Gridap to hide the cached vector from the user of this function.

Reply:
Done in 1685e4d99808277dba2e2f7062deb14e6f969bdb